### ISSUE TITLE
PostgreSql "running post install step…the Database Cluster Initialisation failed"

### ISSUE THREAD

**Issue by mohamkz** [EnterpriseDB/edb-installers#220]
![img](https://github.com/user-attachments/assets/e5b67f64-d5fa-4e8f-9d6b-2a24dbb713d3)
I keep having this problem with different versions of postgreSQL (17 - 16 - 15)
[Uploading install-postgresql.log…]()
this is the log file

**Comment by merwusch** [comment]
Same here

**Comment by Nonax71** [comment]
Same here

**Comment by sandeep-edb** [comment]
Please share the installation log to help us understand the cause better

**Comment by Nonax71** [comment]
I think I found the problem with the installation on my (WIN10) system. The installer seems to use a VBS script (initcluster.vbs), but for security reasons this VBS scripts are disabled on my system. I was able to install Postgres on another system (WIN11)

**Comment by sandeep-edb** [comment]
Yeah, we are migrating those legacy scripts to the new technology. Hopefully, they should be done soon.

**Comment by Slowly-Grokking** [comment]
> I think I found the problem with the installation on my (WIN10) system. The installer seems to use a VBS script (initcluster.vbs), but for security reasons this VBS scripts are disabled on my system. I was able to install Postgres on another system (WIN11)

thanks for troubleshooting that. I was able to install by temporarily enabling in regedit:
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows Script Host\Settings
HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\Windows Script Host\Settings

**Comment by michae1michae1** [comment]
I am getting the same error on Win 11 - apparently, I am not allowed to share install log (from my company)

**Comment by amruthamksree** [comment]
same problem for me also , any solution for this ? can anyone help me on this ? Thanks, in advance.

**Comment by Slowly-Grokking** [comment]
> same problem for me also , any solution for this ? can anyone help me on this ? Thanks, in advance.

Use Regedit and and change these from enabled = 0, to enabled = 1.  

HKEY_CURRENT_USER\Software\Microsoft\Windows Script Host\Settings
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows Script Host\Settings
HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\Windows Script Host\Settings.

Then reboot and run the installer again, and you should be good to go. Also disable them again in regedit after you finish.

**Comment by nnkoc** [comment]
when i try 15 times this is correct congratulations [Slowly-Grokking](https://github.com/Slowly-Grokking)

**Comment by Manika-EDB** [comment]
Hi,

Can you please test this **test/snapshot installer** and let us know how is it going?
https://get.enterprisedb.com/test-installers/postgresql-17.2-0.0snapshot13154625551.1107.1.f6efa78.3-windows-x64.exe

This installer is moved away from using any VB Script. Once you confirm the issue is resolved, this fix will be added to the upcoming maintenance releases.

Thanks!

**Comment by Nonax71** [comment]
Hi,

Unfortunately that didn’t work. See attached log.

Kind regards,
Marien

**Comment by Manika-EDB** [comment]
Hi @Nonax71,

I couldn't find the installation log you mentioned. Can you please share it again?

Thanks!

**Comment by Nonax71** [comment]
[install-postgresql.log](https://github.com/user-attachments/files/18873123/install-postgresql.log)

**Comment by Manika-EDB** [comment]
Thank you for sharing the log @Nonax71 

We will look into it and get back to you soon.

**Comment by Manika-EDB** [comment]
Hi @Nonax71,

Can you please try installation with the latest released installers, and let us know if the issue still persist?
You can download the released installers from our official website.

Please let us know if need any further assistance.

Thanks!

**Comment by hjampani1567** [comment]
I am encountering the same post installation error with version 16.8.  Please advise me the next steps.

**Comment by hjampani1567** [comment]
Please find the installation log. 

[install-postgresql.log](https://github.com/user-attachments/files/19761061/install-postgresql.log)

**Comment by Manika-EDB** [comment]
Hi @hjampani1567,

Can you please try to execute this command on your CMD (Run as administrator) and share the output to understand the issue?

`"C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -ExecutionPolicy Bypass -File "D:\Program Files\PostgreSQL\16/installer/server/initcluster.ps1" "NT AUTHORITY\NetworkService" "postgres" "****" "C:\Users\QLIKAD~1\AppData\Local\Temp\2/postgresql_installer_48865077aa" "D:\Program Files\PostgreSQL\16" "D:\Program Files\PostgreSQL\16\data" 4432 "DEFAULT" 0`

Thanks!

**Comment by hjampani1567** [comment]
Thanks Manika!

Never mind I was able to figured out my issue. The problem seems to be we should uninstall every DB instance residing on the machine and install 16.  While installing we selected Locale as English and no issues to be reported.

Thanks!

**Comment by tomupetra** [comment]
I had the same issue. Does anyone has the solution?
